src/renderer/routes/test_sequencer_panel/models/models.ts

```diff
@@ -27,24 +27,23 @@ export const testSequenceStopRequest: (
   return {
     event: "stop",
     data: tree,
-    hardware_id: null,
-    project_id: null,
   };
 };
 
-export const testSequenceExportCloud: (
+export const testSequenceResumeRequest: (
   tree: TestRootNode,
-  hardware_id: string,
-  project_id: string,
-) => TestSequenceRun = (
+) => TestSequenceRun = (tree: TestRootNode) => {
+  return {
+    event: "resume",
+    data: tree,
+  };
+};
+
+export const testSequencePauseRequest: (
   tree: TestRootNode,
-  hardware_id: string,
-  project_id: string,
-) => {
+) => TestSequenceRun = (tree: TestRootNode) => {
   return {
-    event: "export",
+    event: "pause",
     data: tree,
-    hardware_id: hardware_id,
-    project_id: project_id,
   };
 };
```

Comment:
these functions feel useless, it seems like less typing to just type out the object literal (`{event: type, data: tree}`) instead of importing these functions and calling them, it should be typesafe either way?

Reply:
Typing problem otherwise since the socket can take more than a single type of message

Reply:
We could expose a function just to send this type of msg tho. I'll look at it tmr while updating the `sequencer.context.ts`